bioconda_utils/pkg_test.py

```diff
@@ -99,9 +99,9 @@ def test_package(
     name_override : str
         Passed as the --name-override argument to mulled-build
 
-    channels : None | str | list
-        The local channel of the provided package will be added automatically;
-        `channels` are channels to use in addition to the local channel.
+    channels : list
```

Comment:
I don't think the `None | str` option is used anywhere, so I think it's fine to remove them.